.github/workflows/production.yml

```
@@ -307,10 +307,18 @@ jobs:
           for migration in "${required_migrations[@]}"; do
             remote_tests+="test -s '$REMOTE_RELEASE/tools/migrations/$migration' && "
           done
-          remote_tests=${remote_tests% && }
-          ssh nvx-prod "$remote_tests"
+          if [ -n "$remote_tests" ]; then
+            remote_tests=${remote_tests% && }
+            ssh nvx-prod "$remote_tests"
+          fi
+          echo "PRODUCTION_MIGRATION_PAYLOAD=PASS required=${#required_migrations[@]} exact_sha=$CANDIDATE_SHA"
+=======
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)